src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+    font-family: Avenir;
+    src: url(./fonts/Avenir-Book.ttf);
+    font-weight: 300;
+  }
+  
+  @font-face {
+    font-family: Avenir;
+    src: url(./fonts/Avenir-Heavy.ttf);
+    font-weight: bold;
+  }
+  
+  body {
+    box-sizing: border-box;
+    font-family: Avenir, sans-serif;
+  }
+  
+  .search__form {
+    position: relative;
+    margin-top: 20px;
+  }
+  
+  .search__label::before {
+    content: "";
+    position: absolute;
+    background-image: url(./images/Search.svg);
+    left: 26px;
+    top: 25px;
+    width: 19px;
+    height: 19px;
+  }
+  
+  .search__label--small::before {
+    left: 13px;
+    top: 15px;
+    width: 11px;
+    height: 11px;
+    background-size: contain;
+  }
+  
+  .search__input {
+    font-family: Avenir, sans-serif;
+    width: 100%;
+    border-radius: 4px;
```

Comment:
move `font-family: Avenir, sans-serif` to.html
use inherit for Inheritance font-family from HTML
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit